.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Create Release Pull Request'
+
+on:
+  create:
```

Comment:
Only when the branch is created. If it already exists we don't want to update the PR (since we probably already added info about feature flags, stats, etc.)